libsolidity/formal/VariableUsage.h

```diff
@@ -41,6 +41,8 @@ class VariableUsage: private ASTConstVisitor
 	void setCurrentContract(ContractDefinition const& _contract) { m_currentContract = &_contract; }
 	void setCurrentFunction(FunctionDefinition const& _function) { m_currentFunction = &_function; }
 
+	FunctionDefinition const* getCurrentFunction() { return m_currentFunction; }
```

Comment:
```suggestion
	FunctionDefinition const* currentFunction() { return m_currentFunction; }
```

Reply:
This method is not needed anymore. The issue is resolved in a different way.